types/selenium-webdriver/test/bidi/logInspector.ts

```diff
@@ -0,0 +1,27 @@
+import * as webdriver from "selenium-webdriver";
+import LogInspector from "selenium-webdriver/bidi/logInspector";
```

Comment:
```suggestion
import LogInspector = require("selenium-webdriver/bidi/logInspector");
```

Reply:
i would not have guessed that worked without esModuleInterop, thank you